### PR TITLE
[DOCS-7468] Development Configuration variables

### PR DESCRIPTION
**Pull request by ariadnags**

### CODE REVIEW

**Review by Artur-Piechuta — APPROVED** [review]
LGTM

File: process-automation/latest/admin/release.md [hunk 1]
```diff
@@ -255,3 +255,7 @@ To view the logs:
 4. Select **Logs** and then the logs you want to view.
 
 The log text is color coded. White indicates no issue. Yellow indicates a warning, and Red indicates an error.
+
+### Development Configuration
```
Comment (anxumalo) [inline process-automation/latest/admin/release.md:259]:
```suggestion
### Development configuration
```

**Review by anxumalo — COMMENTED** [review]
Is the UI "Development configuration" or "Development Configuration" ?

**Comment by ariadnags** [comment]
> Is the UI "Development configuration" or "Development Configuration" ?

It's "Development configuration".

**Review by anxumalo — APPROVED** [review]
LGTM 👍🏼

**Pull request merged**